package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@arcadia-eng/utility-connect-react",
-  "version": "2.0.2",
+  "version": "3.0.0",
```

Comment:
Yeah so I know we were talking in chat about this ^^ and some of the changes here are breaking changes

i.e. renaming `ready`-> `loading`

I know last time I bumped this version to 2.0 we had a discussion on whether or not it should match the version in `utility-connect`.

Im thinking the requirement that they match is going to be untenable -- but at the same time it will be weird to have the version mismatch. Just wanted to make the observation, but I think this is fine.

Reply:
Bumped up to version 3.0!